tests/foreman/ui/test_adusergroups.py

```diff
@@ -157,3 +160,124 @@ def test_create_external_adusergroup_3(self):
         @Status: Manual
 
         """
+
+    @stubbed()
+    def test_delete_external_adUserGroup(self):
+        """@Test: Delete External AD User Group
+
+        @Feature: LDAP Authentication - Active Directory - delete
+
+        @Steps:
+
+        1. Create an UserGroup.
+        2. Assign some roles to UserGroup.
+        3. Create an External AD UserGroup as per the UserGroup name in AD.
+        4. Delete the External AD UserGroup
```

Comment:
there are a few instances of missing/inconsistent periods in this PR. Nothing that is a blocker, just nitpicking :)